packages/vue/src/components/molecules/SfRange/SfRange.stories.js

```diff
@@ -0,0 +1,181 @@
+import { SfRange } from "@storefront-ui/vue";
+export default {
+  title: "Components/Molecules/Range",
+  component: SfRange,
+  argTypes: {
+    value: {
+      control: {
+        type: "array",
+      },
+      table: {
+        category: "Props",
+      },
+      defaultValue: [0, 1],
+    },
+    disabled: {
+      control: "boolean",
+      table: {
+        category: "Props",
+      },
+      description: "Disabled",
+    },
+    start: {
+      control: {
+        type: "array",
+      },
+      table: {
+        category: "Props",
+      },
+      defaultValue: [0, 1],
+    },
+    range: {
+      control: "object",
+      table: {
+        category: "Props",
+      },
+      defaultValue: {
+        min: 0,
+        max: 10,
+      },
+    },
+    step: {
+      control: "number",
+      table: {
+        category: "Props",
+      },
+      defaultValue: 1,
+    },
+    connect: {
+      control: "boolean",
+      table: {
+        category: "Props",
+      },
+      defaultValue: true,
+    },
+    direction: {
+      control: "text",
+      table: {
+        category: "Props",
+      },
+      defaultValue: "ltr",
+    },
+    orientation: {
+      control: "text",
+      table: {
+        category: "Props",
+      },
+      defaultValue: "horizontal",
+    },
+    behaviour: {
+      control: "text",
```

Comment:
also here would be good to know what options are available :) maybe we can add descriptions for props with more options to choose :) or link to docs?